### PR TITLE
sssd: update to 2.8.2

### PR DESCRIPTION
**Pull request by klarasm**
Continues from #40846

#### Testing the changes
- I tested the changes in this PR: **YES**
- Also tested with #41948

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for these architectures:
  - aarch64
  - armv7l
  - armv6l
  - i686

This package is not supported on musl.

### CODE REVIEW

**Comment by klarasm** [comment]
I may look into adding a build option to the openssh package to enable integration with sudoers rules on ldap via this package later.

**Comment by klarasm** [comment]
The original author has requested that I reset the commit author, so I did so.

File: srcpkgs/sssd/template [hunk 1]
```diff
@@ -1,50 +1,52 @@
...
-
-do_check() {
-	export CK_TIMEOUT_MULTIPLIER=10
-	make check VERBOSE=yes
-	unset CK_TIMEOUT_MULTIPLIER
-}
+export PYTHON_CONFIG="${XBPS_CROSS_BASE}/usr/bin/python3-config"
 
 post_install() {
 	rm -rf ${DESTDIR}/etc/rc.d
```
Comment (paper42) [inline srcpkgs/sssd/template:52]:
```suggestion
	rm -r ${DESTDIR}/etc/rc.d
```

File: srcpkgs/sssd/template [hunk 2]
```diff
@@ -1,50 +1,52 @@
...
 shlib_provides="libsss_util.so libsss_crypt.so libsss_debug.so"
+# * test_sysdb_sudo: Could not run test: 0 != 0x1 (test_sudo_convert_time);
+# * test_inotify: [test_timeout] (0x0010): The test timed out!
+# * dlopen-tests: Error opening sss.so: [...] conf_get_str: symbol not found
+# * strtonum-tests: Tests:test_strtoint32_alphaonly_base_10:0: errno
+#   unexpectedly set to 22
+# * pam-srv-tests: https://github.com/SSSD/sssd/issues/5631
+# * requires to manually 'make test_CA' (although it should be done by
+#   default?).
+make_check=no
```
Comment (paper42) [inline srcpkgs/sssd/template:47]: I don't think it makes sense to have 2 patches that fix 2 tests when we disable all of them anyway.

Comment (klarasm) [reply srcpkgs/sssd/template:47]: I removed the patches. Tried to look more into the tests that were failing but besides a few of them I don't have any idea how to fix them.

**Pull request merged**